.github/workflows/super-linter.yml

```diff
@@ -19,20 +19,21 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Lint code base
-        uses: docker://github/super-linter:v2.2.2
+        uses: github/super-linter@v4
         env:
-          # Listed but disabled linters would be nice to have.
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
FYI: setting `GITHUB_TOKEN` was needed for listing each test:

<img width="501" alt="Screenshot" src="https://user-images.githubusercontent.com/14186207/214385477-102182f5-d6c5-4b55-bd9b-690e3d8c58ce.png">

Reply:
Oh, I see. I assume you have updated this?

Reply:
Only the markdown files with #2762. The others were apparently good.

Reply:
Before this upgrade of super-linter (incl. GITHUB_TOKEN), it **only** looked like:

<img width="501" alt="Screenshot2" src="https://user-images.githubusercontent.com/14186207/214410265-5b5db3b6-aabd-4967-8e39-9f7d2ec5f19b.png">

which wasn't very informative.

Reply:
> FYI: setting `GITHUB_TOKEN` was needed for listing each test:

Ah, you meant adding the `GITHUB_TOKEN`  line in the super-linteryml file, right? (I initially thought that you have set the secret somewhere in the GH settings.)

Reply:
See beginning of https://github.com/github/super-linter#how-to-use

> NOTE: If you pass the Environment variable GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} in your workflow, then the GitHub Super-Linter will mark the status of each individual linter run in the Checks section of a pull request. Without this you will only see the overall status of the full run. There is no need to set the GitHub Secret as it is automatically set by GitHub, it only needs to be passed to the action.